lib/apiservers/portlayer/swagger.yml

```diff
@@ -1082,12 +1082,14 @@ definitions:
     properties:
       containerId:
         type: string
-      imageId:
+      layerId:
         type: string
-      created:
+      repoName:
         type: string
-        format: datetime
-      name:
+      created:
+        type: integer
+        format: int64
```

Comment:
That's the unix time -- it's placed in the vmx file on container vm creation.  Using time.Unix(n,n) you can covert to time.  i.e. https://play.golang.org/p/Dx2asGut5Y